redundant-bmc/src/manager.cpp

```diff
@@ -22,8 +25,15 @@ Manager::Manager(sdbusplus::async::context& ctx,
 // NOLINTNEXTLINE
 sdbusplus::async::task<> Manager::startup()
 {
+    co_await sibling->init();
+
     ctx.spawn(doHeartBeat());
 
+    if (sibling->isBMCPresent())
```

Comment:
Don't we need to wait for the sibling BMC to be detected if it's not present? I believe both BMCs initialize in parallel, correct? Are you planning to handle this error separately?

Reply:
There's going to be a GPIO for hardware presence that says if the BMC card is even plugged in.  So if the card isn't plugged in, there's no point in calling waitForSiblingUp.

If that other sled is plugged in and the sibling does eventually come up before we power on, the plan is to enable redundancy at that point.